lib/active_force/sobject.rb

```diff
@@ -153,9 +155,10 @@ def persisted?
     end
 
     def self.field field_name, args = {}
+      options = args.reject { |arg| FIELD_OPTIONS.include?(arg) }
```

Comment:
Would this work instead?

```ruby
options = args.except(*FIELD_OPTIONS)
```

Also, I'm not sure if we really need that constant, since this is probably the only place it will be referenced, it's only a few symbols, and I think it's unlikely to change.  But maybe I'm missing some other consideration.

If we do remove `FIELD_OPTIONS`, that line could be the following, which seems a little more readable (in my opinion):

```ruby
options = args.except(:as, :from, :sfdc_name)
```